scanner/config/config.go

```diff
@@ -80,7 +80,9 @@ type IndexerConfig struct {
 	// Enable if false disables the Indexer service.
 	Enable bool `yaml:"enable"`
 	// GetLayerTimeout timeout duration of GET requests for layers
-	GetLayerTimeout Duration `yaml:"get_layer_timeout"`
+	GetLayerTimeout Duration               `yaml:"get_layer_timeout"`
+	Repo            map[string]interface{} `yaml:"repo,omitempty" json:"repo,omitempty"`
+	Package         map[string]interface{} `yaml:"package,omitempty" json:"package,omitempty"`
```

Comment:
Same as in the config sample, `Package` appears unused.

Reply:
Good catch. I saw the errors in the logs when I intentionally provide an invalid repo2cpe_mapping_url but there's no logs related to missing name2repos_mapping_url. I may need to test more.